lending_pool/src/lib.rs

```diff
@@ -89,32 +88,16 @@ pub trait LendingPool:
         self.require_non_zero_address(&initial_caller)?;
 
         let borrow_token_pool_address = self.get_pool_address_non_zero(&asset_to_borrow)?;
-        let lend_token_pool_address = self.get_pool_address_non_zero(&payment_lend_id)?;
+        let ltv = self.get_ltv_exists_and_non_zero(&collateral_token_id)?;
```

Comment:
I'd rename this variable to `loan_to_value`.

Reply:
done